integration-cli/docker_cli_events_unix_test.go

```diff
@@ -56,6 +56,7 @@ func (s *DockerSuite) TestEventsRedirectStdout(c *check.C) {
 
 func (s *DockerSuite) TestEventsOOMDisableFalse(c *check.C) {
 	testRequires(c, DaemonIsLinux)
+	testRequires(c, NativeExecDriver)
```

Comment:
these can't run on lxc for the lstat error above in my other comment. Skipping them for now since maybe someone fix the lstat error